projects/core/src/features-config/utils/feature-config-utils.ts

```diff
@@ -5,6 +5,15 @@ function isFeatureConfig(config: any): config is FeaturesConfig {
 }
 
 function isInLevel(level, version) {
+  if (level === '*') {
+    return true;
+  }
+  if (
+    version.indexOf('!') !== -1 &&
+    version.substr(1, version.length) === level
```

Comment:
@dunqan What was the intention behind the `!` mark? I recall it was `lower than ...` (i.e. `!1.2` should mean `all versions lower than 1.2`). 
But I need to admit that for me it's not super-intuitive that `!` means `lower than`. ;)

Reply:
1. This logic should be moved to `isFeatureLevel` and `isFeatureEnabled` IMO.

3. `!` means negation, so not `lower than'` but like `notFeatureLevel` and `IsFeatureDisabled`.